packages/flutter/test/material/paginated_data_table_test.dart

```diff
@@ -786,4 +786,34 @@ void main() {
     await tester.pumpWidget(buildTable(false));
     expect(find.byType(Checkbox), findsNothing);
   });
+
+  testWidgets('Table has no decoration with DataTableTheme', (WidgetTester tester) async {
```

Comment:
Reword this to say something like `Table should not use decoration from DataTableTheme`

Reply:
Done!